apps/omg_eth/lib/omg_eth/wait_for.ex

```diff
@@ -39,7 +39,7 @@ defmodule OMG.Eth.WaitFor do
   NOTE: `eth_receipt` takes txhash as raw decoded binary, like the rest of Eth APIs, but binaries in the receipt
   returned are in `0xhex-style`
 
-  This is low-level, consider using `|> Eth.DevHelpers.transact_sync!()` for eth-transactions' syncronicity in tests
+  This is low-level, consider using `|> Support.DevHelper.transact_sync!()` for eth-transactions' syncronicity in tests
```

Comment:
this module should be moved into `support` as well, it's not used anywhere else.